src/library/dtls.cpp

```diff
@@ -80,6 +81,57 @@ static void HandleMbedtlsDebug(void *, int level, const char *file, int line, co
     }
 }
 
+/**
+ * This function convert mbedtls error to OT Commissioner error.
+ *
+ * For the implementation details, please reference to <mbedtls/error.h>.
+ *
+ */
+static Error ErrorFromMbedtlsError(int aMbedtlsError)
```

Comment:
Is it possible to preserve the original `aMbedtlsError` in Error?

Reply:
I don't think it is easy to implement with refactoring the `Error` class. By the way, we have already refactored `Error` to include a error message and in the case of mbedtls, it includes the string form of `aMbedtlsError`.